src/app/qgsmaptoolselectradius.cpp

```diff
@@ -205,6 +201,9 @@ void QgsMapToolSelectRadius::keyReleaseEvent( QKeyEvent *e )
 
 void QgsMapToolSelectRadius::updateRubberband( const double &radius )
 {
+  if ( !mRubberBand )
```

Comment:
this will be called if updateRubberBand is called before canvasMoveEvent

Reply:
That's exactly the condition that this PR fixes - previously entering a value, triggering updateRubberBand before a mouse move would result in a crash.